website/src/css/_colors.scss

```diff
@@ -117,3 +117,8 @@ $theme-colors: (
     'cornflower-blue': $cornflower-blue,
     'dodger-blue': $dodger-blue
 );
+
+// TODO: After Tailwind migration this class can be deleted
+.brightness-0 {
+    filter: brightness(0);
+}
```

Comment:
I think we should move this to `styles.scss` under it's own comment section since it's a filter and not directly related to colours

Reply:
Sure thing!